<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mail UCÉ</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mail UCÉ</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrarse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signInButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
